### ISSUE TITLE
"Indent switches" is not always working in Sublime Text 3

### ISSUE THREAD

**Issue by HybridRbt** [akof1314/CoolFormat#11]
This option is not working when used in "Selected format" command, but works fine in "Quick format" command. I was using Sublime Text version 3066 on Windows 7.

**Comment by akof1314** [comment]
I tested was successful. Do you use the parameters (-S) see http://akof1314.github.io/CoolFormat/doc/Cpp.html

eg.

> [SynTidy]
> C++=""-A1-p-N-Y-k3-S""

**Comment by HybridRbt** [comment]
Yes I did. Otherwise it wouldn't be working in Quick Format command, right?
And this is my parameter list:
C++=""-A1-s4-C-S-fpH-N-Y-k3-y""

Did I form it correctly?

Thanks,
Jianfeng

**Comment by akof1314** [comment]
Can you show some code that not working when used in "Selected format" command?

**Comment by HybridRbt** [comment]
I am sorry for not getting back to you in time. I was just applying the
command to the example you provided in your document. I have attached
several pictures to show you the config of my coolformat settings and the
format before/after the command was applied.

Thanks,
Jianfeng

**Comment by akof1314** [comment]
I can not see the attached picture...
Can you show me at email kof1234##sina.com

**Comment by HybridRbt** [comment]
Sure, I will forward this email to your inbox at sina. Or if you prefer, I
can post them on github discuss form. Your call. :)

**Comment by akof1314** [comment]
When you use "Selected format" command.
you should select function

```
void abc()
{
    switch(foo)
    {case 1:a += 1;
     break; case 2:
    {
            a += 2;
            break;
    }
    }
}
```

instead of

```
    switch(foo)
    {case 1:a += 1;
     break; case 2:
    {
            a += 2;
            break;
    }
    }
```

**Comment by HybridRbt** [comment]
Thanks for pointing that out! I have tried as you suggested, but it still
failed. I have attached the pictures.

WuHuan notifications@github.com于2015年4月24日周五 上午10:28写道：

**Comment by akof1314** [comment]
I tested was successful. I send you email
